docs/authentication.md

```diff
@@ -0,0 +1,303 @@
+```
+
+#### Game Center
+
+Apple devices have builtin authentication which can be done without user interaction through Game Center. The register or login process is a little complicated because of how Apple's services work.
+
+```csharp fct_label="Unity"
+// You'll need to use native code (Obj-C) with Unity.
+// The "UnityEngine.SocialPlatforms.GameCenter" doesn't give enough information
+// to enable authentication.
+
+// We recommend you use a library which handles native Game Center auth like
+// https://github.com/desertkun/GameCenterAuth
+
+string playerId = "...";
+string bundleId = "...";
+string base64salt = "...";
+string base64signature = "...";
+string publicKeyUrl = "...";
+long timestamp = 0L;
+
+var message = NAuthenticateMessage.GameCenter(
+    playerId, bundleId, timestamp, base64salt, base64signature, publicKeyUrl);
+client.Register(message, (INSession session) => {
+  Debug.LogFormat("Session: '{0}'.", session.Token);
+}, (INError err) => {
+  Debug.LogErrorFormat("Error: code '{0}' with '{1}'.", err.Code, err.Message);
+});
+// Use client.Login(...) after register.
+```
+
+#### Steam
+
+Steam requires you to configure the server before you can register a user.
+
+!!! Note "Server configuration"
+    Have a look at the [configuration](install-configuration.md) section for what settings you need for the server.
+
+```csharp fct_label="Unity"
+string sessionToken = "...";
+
+var message = NAuthenticateMessage.Steam(sessionToken);
+client.Register(message, (INSession session) => {
+  Debug.LogFormat("Session: '{0}'.", session.Token);
+}, (INError err) => {
+  Debug.LogErrorFormat("Error: code '{0}' with '{1}'.", err.Code, err.Message);
+});
+// Use client.Login(...) after register.
+```
+
+### Custom
+
+A custom identifier can be used in a similar way to a device identifier to login or register a user. This option should be used if you have an external or custom user identity service which you want to use. For example EA's Origin service handles accounts which have their own user IDs.
+
+A custom identifier must contain alphanumeric characters with dashes and be between 10 and 60 bytes.
+
+```csharp fct_label="Unity"
+// Some id from another service.
+string customId = "a1fca336-7191-11e7-bdab-df34f6f90285";
+
+var message = NAuthenticateMessage.Custom(customId);
+client.Register(message, (INSession session) => {
+  Debug.LogFormat("Session: '{0}'.", session.Token);
+}, (INError err) => {
+  Debug.LogErrorFormat("Error: code '{0}' with '{1}'.", err.Code, err.Message);
+});
+// Use client.Login(...) after register.
+```
+
+## Sessions
+
+The register and login messages return a session on success. The session contains the current user's ID and handle as well as information on when it was created and when it expires.
+
+!!! Tip
+    You can change how long a session is valid before it expires in the [configuration](install-configuration.md) in the server. By default a session is only valid for 60 seconds.
```

Comment:
"By default a session token is only ..." 

Without the word "token" - it might come across that the server only holds the connection active for 60seconds.